utils/getUniqueFields.js

```diff
@@ -0,0 +1,9 @@
+const getUniqueFields = (message) => {
+  const content = message.match(/\{([^}]+)\}/)[1]
+
+  const keys = content.match(/\b(\w+)(?=:)/g).join(', ')
+
+  return keys
```

Comment:
maybe return content.match without adding it to "keys" variable at once

Reply:
I thought about it, but decided to leave it like this to be more clear what exactly this function returns